site/pages/create-social-media-post/[disruptionId]/[socialMediaPostIndex].page.tsx

```diff
@@ -240,7 +240,7 @@ export const getServerSideProps = async (ctx: NextPageContext): Promise<{ props:
     return {
         props: {
             ...getPageState(errorCookie, socialMediaPostSchema, disruptionId, socialMediaPost || undefined),
-            disruptionSummary: disruption?.summary || "",
+            disruptionDescription: disruption?.summary || "",
```

Comment:
This needs changing to disruption?.description to copy the description

Reply:
done